projects/mercury/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```

Comment:
Could you also push the updated the [gradle-wraper.jar](https://github.com/thehyve/sils-fairspace/blob/cf1a8a7e7a85a3192f6f911ac768eba9f91ce357/projects/mercury/gradle/wrapper/gradle-wrapper.jar) for the new version?